drivers/npmx/shell/shell_common.c

```diff
@@ -8,9 +8,6 @@
 #include <zephyr/kernel.h>
 #include <npmx_driver.h>
 
-/** @brief Max supported number of shell arguments. */
-#define SHELL_ARG_MAX_COUNT 3U
-
```

Comment:
`SHELL_ARG_MAX_COUNT` should be put in header file in the first commit in the first place, instead of moving it in ADC commit.

Reply:
Bump. Now it is both in header and in source file in the first commit.